style.css

```diff
@@ -0,0 +1,922 @@
+  }
+
+  .nav strong img {
+    height: 2.6rem;
+    margin-top: 0.5rem;
+  }
+
+  .nav button {
+    color: #ec5242;
+    border: 3px solid #ec5242;
+    background-color: #fff;
+    height: 2.8rem;
+    font-family: var(--font-family);
+    font-size: 14px;
+    margin-top: 0.5rem;
+    padding: 0.5rem 1rem 0.5rem 1rem;
+  }
+
+  .click {
+    text-decoration: none;
+    color: #272a31;
+  }
+
+  .click:hover {
+    color: #ec5242;
+  }
+
+  .back {
+    background: url("images/1678984284631.jpg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 740px;
+  }
+
+  .pad {
+    margin: 4rem 10rem 7rem 10rem;
+  }
+
+  .pad h2 {
+    color: #ec5242;
+    font-size: 33px;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  }
+
+  .pad h1 {
+    font-size: 48px;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    background-image: url("images/pexels-pixabay-267151.jpg");
+    background-size: contain;
+    background-position: top center;
+    -webkit-background-clip: text;
+    color: transparent;
+    margin-top: -1rem;
+  }
+
+  .box {
+    font-family: var(--font-family);
+    font-size: 25px;
+    width: 60vw;
+    padding: 2rem;
+    border: 3px solid #fff;
+    color: #272a31;
+  }
+
+  .pad h3 {
+    font-size: 35px;
+    font-family: var(--font-family);
+    color: #515151;
+  }
+
+  footer p {
+    font-family: var(--font-family);
+    margin-left: 2rem;
+    font-style: italic;
+  }
+
+  .pad p {
+    font-size: 27px;
+    font-family: var(--font-family);
+    color: #868686;
+  }
+
+  .main {
+    display: grid;
+    place-items: center;
+    background-image: url("images/abstract-background-lineart-cubes_95169-324.jpg");
+    background-size: cover;
+    height: 490px;
+    padding: 1rem 3rem 1rem 3rem;
+  }
+
+  .main h2 {
+    color: #fff;
+    font-family: var(--font-family);
+    font-size: 27px;
+  }
+
+  .line {
+    background-color: #ec5242;
+    width: 3rem;
+    height: 2px;
+    margin-top: -6rem;
+  }
+
+  .big-box {
+    display: flex;
+    font-family: var(--font-family);
+    margin-top: -3rem;
+  }
+
+  .big-box .inner-box {
+    margin: 3px;
+    background-color: rgba(247, 246, 243, 0.2);
+    width: 10rem;
+    padding: 0.6rem;
+  }
+
+  .big-box .inner-box:hover {
+    border: #fff 2px solid;
+  }
+
+  .end-box strong img {
+    height: 2.6rem;
+    margin-top: 0.5rem;
+  }
+
+  .partner ul li img {
+    height: 3rem;
+    margin: 0 1rem 0 1rem;
+  }
+
+  .big-box .inner-box img {
+    margin-left: 3rem;
+  }
+
+  .big-box .inner-box h4 {
+    color: #ec5242;
+    font-size: 20px;
+    text-align: center;
+    margin-top: -0.1rem;
+  }
+
+  .main p {
+    color: #fff;
+    font-size: 18px;
+    font-family: var(--font-family);
+    margin-top: -1rem;
+  }
+
+  .main .underline:hover {
+    text-decoration: underline;
+  }
+
+  .feature p {
+    color: #000;
+    font-family: var(--font-family);
+    font-size: 14px;
+  }
```

Comment:
- [ ] You have done incredibly well to implement the speaker's section dynamically 👏. However, we need to ensure the design matches the original design. For this case, we need to change the text color to match the original design. In addition, ensure the text does not overlap with the image of another speaker and the section below. You can achieve this by ensuring the margin is similar to the one in the original design ☺️.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/226078559-b9d9450e-2c64-4798-bf30-74a999d8e8bb.png)

Original design 😎 
![image](https://user-images.githubusercontent.com/99394374/226078594-2243b5fb-70f9-4343-aca5-4422feb0b88f.png)